### PR TITLE
Rescue from MultipleIncludedBlocks when attempting to require dependencies

### PR DESCRIPTION
**Pull request by philbirt**
Since upgrading to Rails 4.2, if a dependency with ActiveSupport::Concern is loaded multiple times, an exception is raised. I think we just want to ignore these errors, since the file has been loaded.

### CODE REVIEW

**Comment by athal7** [comment]
i don't completely follow here. could double loading that code potentially cause issues? why is it happening?

**Comment by philbirt** [comment]
tldr; I'm going to close this out, as it is just a symptom of a bigger problem with our spree-backend + solidus work (specifically all of our decorators), and not necessarily a factory_girl-preload problem

Essentially if rails loads a concern twice, it now will throw and exception. Since reloading things in app/models/** is not isolated to this gem, but also whenever a change is made locally and your server needs to pull that change, console reload, etc, this needs a different solution, and that solution has to be in spree-backend.

Adam has already seen this issue when porting over a lot of the extensions, and has a pretty simple solution to get out of it  An example of how he fixed signifyd is here: https://github.com/solidusio/solidus_signifyd/commit/5e72e5d7f1615b06f00c165c3a14696ac2c9a2fb

**Comment by athal7** [comment]
cool. thanks for the explanation, that sounds good!
